info/v2/container.go

```diff
@@ -52,6 +52,14 @@ type MemorySpec struct {
 	SwapLimit uint64 `json:"swap_limit,omitempty"`
 }
 
+type ContainerInfo struct {
+	// Describes the container.
+	Spec ContainerSpec `json:"spec,omitempty"`
```

Comment:
Given that we are essentially breaking the existing API, which I think is ok, shall we get rid of some of the fields in `Spec`? I want to get rid of all the `Has*` fields in place of pointer values in Stats.